classes/class-wc-connect-nux.php

```diff
@@ -471,26 +474,22 @@ public function show_banner_before_connection() {
 
 			$jetpack_status = $this->get_jetpack_install_status();
 			$button_text    = __( 'Connect', 'woocommerce-services' );
-			$banner_title   = __( 'Connect your store to activate WooCommerce Services', 'woocommerce-services' );
+			$banner_title   = __( 'Connect Jetpack to activate WooCommerce Services', 'woocommerce-services' );
 			$image_url      = plugins_url( 'images/wcs-notice.png', dirname( __FILE__ ) );
 
 			switch ( $jetpack_status ) {
 				case self::JETPACK_NOT_INSTALLED:
 					$button_text  = __( 'Install Jetpack and connect', 'woocommerce-services' );
-					$banner_title = __( 'Connect Jetpack to activate WooCommerce Services', 'woocommerce-services' );
+
```

Comment:
Accidental extra line?

Reply:
Yep - going to rewrite that commit to remove it.

Reply:
Done in d5617a2.